pkg/restore/restore.go

```diff
@@ -427,6 +428,18 @@ func (ctx *context) restoreFromDir(dir string) (api.RestoreResult, api.RestoreRe
 	return warnings, errs
 }
 
+// objectsAreEqual indentifies whether a backed up object
+// and its representation in the cluster are equal
+func (ctx *context) objectsAreEqual(fromBackup *unstructured.Unstructured, resourceClient client.Dynamic, restoreErr error) (bool, error) {
```

Comment:
I don't think we need to pass in `restoreErr` any more?

Reply:
Sorry for going back and forth on this, but I'm wondering if it would just be clearer to move the `Get()` into the `IsAlreadyExists` block and get rid of this helper. WDYT?

Reply:
My very first pass did that. The reason I named this function `resolveConflict` was because I envisioned adding more complex cases like service accounts there later. That way, the resolution was mostly out of the `IsAlreadyExists` block.

Doing the `Get()` in the block seems reasonable to me, and some function that contains different actions or policy checks added when it's needed.